npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater.rb

```diff
@@ -53,11 +55,56 @@ def updated_dependency_files
           )
         end
 
+        if Experiments.enabled?(:yarn_berry)
+          # There's probably a way to avoid this duplication between the
+          # vendor_updater and install_state_updater, but yeah.
```

Comment:
```suggestion
          # There's probably a way to avoid this duplication between the
          # vendor_updater and install_state_updater, but yeah.
```

😄 FWIW, I think this is fine - I always feel bad when I can't figure out some Ruby golf to compact a statement but this is clear and works so 👍🏻